src/shared/watchdog.c

```diff
@@ -93,15 +77,14 @@ static int ping_watchdog(void) {
 }
 
 
-static int update_timeout(bool enable) {
+static int update_timeout(void) {
```

Comment:
you are basically undoing something here that an earlier commit in this PR added. Would be better to not add that in the first place

Reply:
Ok I'll drop it.